README.md

```diff
@@ -25,7 +25,18 @@ trait DockerKit {
 
 That makes it possible to configure it through enviroment variables
 
-Boot2docker setup
+### docker-machine setup
+
+Docker Toolbox 1.9 (maybe also before, came some time in 2015) contains the [`docker-machine`](https://docs.docker.com/machine/) command. With it, you get going simply by:
+
+```
+docker-machine start default
+eval $(docker-machine env default)
+```
+
+<!-- I like having '$' in shell commands shown in documentation. However, left them out here for consistency with the rest of the README. AKa010216 -->
+
+### Boot2docker setup
```

Comment:
added the sub-sub-titles for consistency (it's really a 1-or-2-or-3 situation here?)